provers/cairo/src/commands.rs

```diff
@@ -13,6 +13,8 @@ pub enum ProverEntity {
     Compile(CompileArgs),
     #[clap(about = "Generate a proof for a given compiled cairo program")]
     Prove(ProveArgs),
+    #[clap(about = "Generate a proof from a given trace of a cairo program execution")]
+    ProveTrace(ProveTraceArgs),
```

Comment:
Let's make a little change of names; I think it is more accurate if the `Prove` command is renamed to something like "RunAndProve" (the command description should be changed too), and the new command you implemented should be the one named `Prove`.
This reflects better the intent of both commands IMO